app/models/course.rb

```diff
@@ -285,6 +285,10 @@ def new_draft_attributes
     )
   }
 
+  scope :engineers_teach_physics, lambda {
+    where(campaign_name: "engineers_teach_physics")
+  }
+
```

Comment:
```suggestion
```

^^ That looks way better

Reply:
https://media.giphy.com/media/SwyTq2jJxc9im6BYnN/giphy.gif